src/main/java/net/geforcemods/securitycraft/items/UniversalOwnerChangerItem.java

```diff
@@ -130,35 +130,22 @@ else if(BlockUtils.getBlock(world, pos.down()) instanceof ReinforcedDoorBlock ||
 			if (hand == Hand.MAIN_HAND && player.getHeldItemOffhand().getItem() == SCContent.BRIEFCASE.get()) {
 				ItemStack briefcase = player.getHeldItemOffhand();
 
-				if (isOwnedBy(player, briefcase)) {
+				if (UniversalKeyChangerItem.isOwnedBy(briefcase, player)) {
```

Comment:
Ok now that I'm reading this, it makes more sense to move the `isOwnedBy` method to BriefcaseItem ^^